Cargo.toml

```diff
@@ -49,6 +50,9 @@ serde_derive = "1.0.0"
 serde = "1.0.0"
 clippy = { version = "=0.0.142", optional = true }
 chrono = "0.4.0"
+pulldown-cmark = { version = "0.0.15", default-features = false }
+ammonia = "0.5.0"
+syntect = "1.7.1"
```

Comment:
If we leave out the patch version, we get the latest patch version when the Cargo.lock file gets updated via `cargo update`.